Update to PHPUnit v6 and remove root namespace slash in use statements

ziadoz opened the pull request:
This is just some prep work before I add some new bits and pieces.

liamkeily commented:
Looks good to me 👍

maxakropolis commented on tests/CategoriesTest.php:
```diff
@@ -1,9 +1,9 @@
-use \DemandwareXml\Assignment;
-use \DemandwareXml\Category;
-use \DemandwareXml\Document;
+use DemandwareXml\Assignment;
```
Should these be grouped?

ziadoz replied:
@maxakropolis I've fixed these.

ziadoz commented:
@fusionspim/devs I've added a PHP CS Fixer configuration and applied the changes it found. I've also added that, plus a PHP syntax check to the build steps too.

weshooper commented:
👍🏻

maxakropolis commented:
👍
